test/view_component/custom_layouts_integration_test.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class CustomLayoutsIntegrationTest < ActionDispatch::IntegrationTest
+  test "test preview index renders custom application layout if configured" do
```

Comment:
Maybe we can avoid rewriting "test" in each test case?

```suggestion
  test "preview index renders custom application layout if configured" do
```